spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -532,4 +525,13 @@
       end
     end
   end
+
+  describe "#subjects_for_scheme" do
```

Comment:
Small unit test. It's going to be a pain to change things so that we do not include a key if it's blank, so for now I'm just returning values that would evaluate true for `#blank?`.